packages/big-design/src/components/Worksheet/Row/styled.ts

```diff
@@ -0,0 +1,9 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled from 'styled-components';
+
+export const StyledTableRow = styled.tr<{ isChild: boolean; isExpanded: boolean }>`
+  display: ${({ isExpanded }) => (isExpanded ? 'visible' : 'none')};
```

Comment:
`visible` isn't a valid `display` attribute. You might want to change it to `table-row`;